src/lib/wicked/adapter.js

```diff
@@ -266,7 +282,17 @@ class WickedAdapter {
      * @return {Promise} Result of the operation
      */
     reloadConnection(name) {
-        return this.client.reloadConnection(name);
+        return new Promise((resolve, reject) => {
+            this.client.reloadConnection(name)
+                    .then((name) => resolve(name))
```

Comment:
You could simply pass the resolve function, but I guess this form is more readable.

Reply:
> but I guess this form is more readable.

Actually not ;)

I prefer your suggestion

```js
.then(resolve)
```

than the current implementation, which is there after some changes from a _multiline resolve_ to the oneliner.

So, thanks!